model/license.go

```diff
@@ -51,8 +52,11 @@ type TrialLicenseRequest struct {
 	ReceiveEmailsAccepted bool   `json:"receive_emails_accepted"`
 }
 
-func (tlr *TrialLicenseRequest) ToJson() string {
-	b, _ := json.Marshal(tlr)
+func (lr *TrialLicenseRequest) ToJson() string {
+	b, _ := json.Marshal(struct {
+		*TrialLicenseRequest
+		IsTrial bool `json:"is_trial"`
+	}{lr, true})
```

Comment:
Decided to hard-code the JSON `"is_trial": true` key/value because there is currently no requirement to request non-trial licenses from Mattermost server.

Reply:
Yes. Good call.

Reply:
@mkraft we may not even need this given that in the CWS endpoint for trial I will explicitly set the flag to true to avoid any requests that can override this with a `false` value.

Reply:
Thanks. I didn't realize this was going through CWS, I thought the request was going directly to the license generator lambda via a DNS entry for customers.mattermost.com. I'll remove this `is_trial` JSON key.